__tests__/endpoints.test.js

```diff
@@ -217,7 +217,7 @@ describe("GET /api/articles", () => {
         expect(articles).toBeInstanceOf(Array);
         expect(articles).toHaveLength(12);
         articles.forEach((article) => {
-          expect(article).toMatchObject({
+          expect(article).toStrictEqual({

```

Comment:
Why did you refactor to this?

Reply:
I was told to make sure the body wasn't being sent in the response & toMatchObject still passed the test, therefore I used toStrictEqual :)

Reply:
sorted